src/command_line_parser.cc

```diff
@@ -1904,57 +1919,64 @@ TritonParser::ParseBackendConfigOption(const std::string& arg)
   return {name_string, setting_string, value_string};
 }
 
-std::tuple<std::string, std::string, std::string>
-TritonParser::ParseGrpcRestrictedProtocolOption(const std::string& arg)
+void
+TritonParser::ParseRestrictedFeatureOption(
+    const std::string& arg, const std::string& option_name,
+    const std::string& key_prefix, const std::string& feature_type,
```

Comment:
nit: it's called `feature_type` in .cc and `feature_name` in .h -- let's make them match for parsing code